app/assets/stylesheets/custom.scss

```diff
@@ -24,6 +24,10 @@ h6 {
   font-weight: 400;
 }
 
+table a, a:hover{
+  color: #A5C6DE;
```

Comment:
Color literals like `#A5C6DE` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#A5C6DE` should be written as `#a5c6de`